### DISCUSSION TITLE
We were not able to save the email channel

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4509] [Self hosted]
Hi, I made a second account in a self hosed installation but when I try to create a new inbox a receive thi errore:
We were not able to save the email channel

![image](https://user-images.githubusercontent.com/62311149/164022587-384416a9-395f-4761-80d6-9b88584b2d00.png)

**Comment by sojan-official** [comment]
@mattiamarras  could you share the server logs if any ?
